renderer/types/peripheral-list.ts

```diff
@@ -1,3 +1,17 @@
-import { Device } from "../../protos/protos";
+import { Param } from "../../protos/protos";
 
-export type PeripheralList = { [uid: string]: Device };
+export interface Peripheral {
```

Comment:
Probably should add a comment for why this is needed instead of the Device proto

Reply:
@AVSurfer123 Good idea